.github/workflows/build.yml

```diff
@@ -109,13 +125,13 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - id: setup-python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           python-version: 3.9
```

Comment:
```suggestion
          python-version: "3.10"
```

Reply:
Nevermind. This was missed in the skeleton so I will PR to fix it there and it will propagate down next kraken.